taxonomy-manager-rest-server/src/test/resources/jwtClaims.json

```diff
@@ -0,0 +1,12 @@
+{
+  "iss": "https://dev.taxman.digirati.com/",
+  "jti": "a-123",
+  "sub": "jdoe-using-jwt-rbac",
+  "upn": "jdoe@quarkus.io",
+  "preferred_username": "jdoe",
+  "aud": "using-jwt-rbac",
```

Comment:
Hm, if the `aud` field is present doesn't match the server the token was sent to (via the `Authorization` header) the implementation is supposed to throw an error. Does Quarkus not complain about this?

Reply:
No, the only ones it seems to care about are `iss` and `groups` atm. Possibly needs a bit of config to make it check that too

Reply:
That makes it non-compliant with RFC 7519, which isn't a huge deal but definitely something to keep in mind. I'll have a look over at the SmallRye JWT implementation.